_posts/2019/2019-01-27-replace-timecop-with-rails-time-helpers-in-rspec.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Replace Timecop With Rails’ Time Helpers in RSpec
+description: You can use the built-in functionality
+layout: article
+category: ruby
+image:
+  base: '2019/replace-timecop-with-rails-time-helpers-in-rspec'
+  alt: 'Jean-Claude Van Damme as Max Walker'
+  source: ''
+  credit: ''
+---
+
+Time-sensitive tests can be a pain.
+
+Thankfully for many years the `timecop` gem served as a way to “freeze” and “time travel” during tests, so that time elapsing during your tests doesn't effect the results.
```

Comment:
```suggestion
Thankfully, for many years the `timecop` gem served as a way to “freeze” time and “time travel” during tests, so that any time that elapsed during your tests running didn't affect the results.
```